### ISSUE TITLE
Update only after answering question to do so?

### ISSUE THREAD

**Issue by timoreichen** [shannah/jdeploy#128]
Now there is "Auto update settings" .. On/Bugs..Never. Is there a way to do it after having asked the user? Like "Hey, there is a new version out. Would like to update now?" If not, the questions arise always on start but the user can choose when to update.

This would be a great feature imho, as you know, it gives the user the feeling of control over the software. But more important, if he is in a hurry and just want to use the application shortly, the user will be thankful not to wait and just postpone it.
